### ISSUE TITLE
[http-server-csharp] some generated files do not have the `#nullable enable` annotation in the file, on purpose?

### ISSUE THREAD

**Issue by ArcturusZhang** [microsoft/typespec#5225]
As titled, some files have this annotation such as those in `lib` directory, but others do not, is this on purpose?

**Comment by markcowl** [comment]
@ArcturusZhang Yes.  We are generating reference properties without nullable by design - in the future we may change this and add the nullable annotation
